src/mbgl/renderer/circle_bucket.hpp

```diff
@@ -28,10 +27,14 @@ class CircleBucket : public Bucket {
     void drawCircles(CircleShader&, gl::Context&);
 
 private:
-    CircleVertexBuffer vertexBuffer_;
-    TriangleElementsBuffer elementsBuffer_;
+    std::vector<CircleVertex> vertexes;
+    std::vector<gl::Triangle<uint16_t>> indexes;
```

Comment:
@kkaefer Should we hard-code `uint16_t` as the index type? So a declaration like this would be:

```C++
std::vector<gl::Triangle> indexes;
```